src/btree/bt_cursor.c

```diff
@@ -1247,7 +1219,7 @@ __btcur_update(WT_CURSOR_BTREE *cbt, WT_ITEM *value, u_int modify_type)
     yield_count = sleep_usecs = 0;
 
     /* It's no longer possible to bulk-load into the tree. */
-    __cursor_disable_bulk(session, btree);
+    __wt_cursor_disable_bulk(session);
```

Comment:
Could you add an assert here and below: `WT_ERR_ASSERT(session, S2BT(session) == btree, WT_PANIC, "blah blah blah")`? I'm keen to make sure we are always disabling bulk on the right thing.

Reply:
Added assert, what do you think of the error message?

Reply:
lgtm - thanks!